projects/canopy/src/lib/modal/modal.service.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { TestBed, waitForAsync } from '@angular/core/testing';
+
+import { spy, verify } from 'ts-mockito';
+import { Subscription } from 'rxjs';
+
+import { LgModalService } from './';
+
+describe('LgModalService', () => {
+  let service: LgModalService;
+  let serviceSpy: LgModalService;
+  const id = 'test-1';
```

Comment:
I'm surprised prettier doesn't re-order these 🤔

Reply:
I'll do it :D

Reply:
Updated